google/auth/_default.py

```diff
@@ -59,6 +59,38 @@
 https://cloud.google.com/docs/authentication/adc-troubleshooting/user-creds. \
 """
 
+_GENERIC_LOAD_METHOD_WARNING = """\
+The {} method is being deprecated because of a potential security risk.
```

Comment:
#QUESTION:

Are we using the right wording OR does the current wording provide the right connotation?

When I first saw the words `is being deprecated` I felt like we were insinuating that this deprecation would take place in the future.

Would this wording be a more assertive explanation?

```suggestion
The {} method is deprecated because of a potential security risk.
```